components/tools/OmeroPy/src/omero/plugins/metadata.py

```diff
@@ -554,6 +565,73 @@ def populateroi(self, args):
                 meas = ctx.get_measurement_ctx(i)
                 meas.parse_and_populate()
 
+    def pixelsize(self, args):
+        "Set physical pixel size"
+        md = self._load(args)
+        client, conn = self._clientconn(args)
+
+        units = {
```

Comment:
This lookup can be achieved via:

```
In [34]: omero.model.LengthI.SYMBOLS.keys()[omero.model.LengthI.SYMBOLS.values().index("mm")]
Out[34]: 'MILLIMETER'
```

so that no hard-coding is needed. You'd have to special case `um`, though, regardless and unicode is going to be challenging from the terminal.